rxbinding-leanback-v17/src/androidTest/java/com/jakewharton/rxbinding/support/v17/leanback/widget/RxSearchBarTestActivity.java

```diff
@@ -0,0 +1,23 @@
+package com.jakewharton.rxbinding.support.v17.leanback.widget;
+
+import com.jakewharton.rxbinding.support.v17.leanback.test.R;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.v17.leanback.widget.SearchBar;
+import android.support.v17.leanback.widget.SearchEditText;
+import android.widget.FrameLayout;
+
+public final class RxSearchBarTestActivity extends Activity {
+  SearchBar searchBar;
+  SearchEditText searchEditText;
+
+  @Override protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.rx_lb_search_fragment);
```

Comment:
I don't think this layout is needed. Just `new SearchBar(this)` and call `setContentView` with that instance.